src/ray/object_manager/object_store_notification.h

```diff
@@ -19,47 +19,45 @@
 
 namespace ray {
 
-// TODO(hme): document public API after refactor.
-class ObjectStoreClient {
+class ObjectStoreNotification {
```

Comment:
Document the class with `\class ObjectStoreNotification` followed by a description. Also, this is sort of a weird name for the class, I think, since it is handling notifications and is not a notification itself. Maybe `ObjectStoreNotificationManager`?